(fix) improve storage.get docs (#2416)
aws-amplify/docs: docs/lib/storage/fragments/js/download.md

- `public`: Accessible by all users of your app. Files are stored under the `public/` path in your S3 bucket.
- `protected`: Readable by all users, but writable only by the creating user. Files are stored under `protected/{user_identity_id}/` where the `user_identity_id` corresponds to the unique Amazon Cognito Identity ID for that user.
    **:**

    ```javascript
    // To get current user's objects
    await Storage.get('filename.txt', { level: 'protected' })
    // To get other users' objects
    await Storage.get('filename.txt', { 
        level: 'protected', 
        identityId: 'xxxxxxx' // the identityId of that user
    })
    ```
- `private`: **Only accessible for the signed in user**. Files are stored under `private/{user_identity_id}/` where the `user_identity_id` corresponds to the unique Amazon Cognito Identity ID for that user.

### Download expiry

You can use `expires` option to limit the availability of your URLs. This configuration returns the pre-signed URL that expires in 60 seconds:

```javascript
await Storage.get('filename.txt', { expires: 60 })
```

> ### Frequent User Questions

Reviewer: User Issues?

> ### Frequent User Questions

Author: ```suggestion
### Frequently Asked Questions
```

> Users often run into unexpected issues, so we are giving you advance notice in documentation with links to open issues - please upvote what you need, to help the team prioritize.

Reviewer: Users can run into

> Users often run into unexpected issues, so we are giving you advance notice in documentation with links to open issues - please upvote what you need, to help the team prioritize.

Author: ```suggestion
Users can run into unexpected issues, so we are giving you advance notice in documentation with links to open issues - please upvote what you need, to help the team prioritize.
```

> Users can run into unexpected issues, so we are giving you advance notice in documentation with links to open issues - please upvote what you need, to help the team prioritize.

Author: ```suggestion
Users can run into unexpected issues, so we are giving you advance notice in documentation with links to open issues - please vote for what you need, to help the team prioritize.
```
